package nextstep.di.factory.exception;

public class BeanInvokeException extends RuntimeException {
    public BeanInvokeException(Throwable cause) {
        super(cause);
    }
}
